docs/cni/rkt/README.md

```diff
@@ -71,32 +71,29 @@ You should also verify each host can access etcd.  The following will return an
 
     curl -L http://localhost:2379/version
 
-And finally check that Docker is running on both hosts by running
-
-    docker ps
-
 
 ## 2. Starting Calico services
 
-Once you have your cluster up and running, start calico on all the nodes
+Once you have your cluster up and running, start Calico on all the nodes.
+Because `rkt` doesn't support daemonizing containers directly, we'll use [systemd-run][systemd-run]
+to create a systemd service.
 
 On calico-01
 
-    sudo calicoctl node
+    sudo systemd-run --unit=calico-node calicoctl node  --runtime=rkt
```

Comment:
Any reason we list this twice? I did a double take to make sure there wasn't something different in the two commands as I ran them...